src/main/java/de/tum/in/www1/artemis/web/rest/vm/LoggerVM.java

```diff
@@ -39,8 +41,30 @@ public void setLevel(String level) {
         this.level = level;
     }
 
+    @Override
+    public int hashCode() {
+        int result = getName() != null ? getName().hashCode() : 0;
+        result = 31 * result + (getLevel() != null ? getLevel().hashCode() : 0);
+        return result;
+    }
+
     @Override
     public String toString() {
         return "LoggerVM{" + "name='" + name + '\'' + ", level='" + level + '\'' + '}';
     }
+
+    @Override
+    public boolean equals(Object object) {
+        if (this == object) {
+            return true;
+        }
+        if (!(object instanceof LoggerVM loggerVM)) {
+            return false;
+        }
+
+        if (!ObjectUtils.nullSafeEquals(getName(), loggerVM.getName())) {
+            return false;
+        }
+        return ObjectUtils.nullSafeEquals(getLevel(), loggerVM.getLevel());
```

Comment:
```suggestion
        if (this == object) {
            return true;
        }

        if (object instanceof LoggerVM loggerVM) {
            return Objects.equals(name, loggerVM.name) && Objects.equals(level, loggerVM.level);
        }

        return false;
```
- `Objects.equals` from the Java standard library should be the same
- fewer negations make this more readable (at least to me)